client/stylesheets/components/_message.scss

```diff
@@ -1,7 +1,11 @@
-.info {
-  background-color: $green;
+.message--success {
+  background: $background-message-success;
 }
 
-.error {
-  background-color: $red;
+.message--info {
+  background-color: $background-message-info;
+}
+
+.message--error {
+  background-color: $background-message-error;
```

Comment:
You might want to use `background` instead of `background-color` for consistency